src/java/fr/paris/lutece/plugins/workflow/modules/formspdf/service/provider/FormsProvider.java

```diff
@@ -0,0 +1,56 @@
+package fr.paris.lutece.plugins.workflow.modules.formspdf.service.provider;
+
+import javax.servlet.http.HttpServletRequest;
+
+import fr.paris.lutece.plugins.forms.service.provider.GenericFormsProvider;
+import fr.paris.lutece.plugins.workflowcore.business.resource.ResourceHistory;
+
+public class FormsProvider extends GenericFormsProvider {
```

Comment:
Comment is needed to explain the context of use of this particular FormsProvider that overrides all default methods